web/src/pages/createStaff/index.page.jsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from 'flowbite-react';
+import React from 'react';
+
+import TextInput from '@/components/organisms/TextInput';
+
+
+//webpage backdround
+const backgroundColor = () => {
+  return (
+    <div className="bg-white min-h-screen p-4"> {/* Use Flowbite class for white background */}
+      
+
+      <form className="mt-10 ml-4 w-1/2 lg:w-1/4  ">
```

Comment:
1. About margin and padding. So much unneccessary classnames, where you can merge it in the div if we have it. 
2. Dont limit the width for now. 
3. The spaces between Inputs can be declared on the wrapper/parent/form component so you dont need to add `mt-5` every single time. Pls utilize it.
```suggestion

      <form className="flex flex-col gap-4">
```

Reply:
sir chev goodevening po .. regarding anii po na codeblock gi follow nakoo ang suggested change po. supposedly ingani po ba ang mahitabo sa mga feild? 
![image](https://github.com/chevyB/pcbee/assets/89174822/24a16da9-11f7-4cfd-9a46-ce2c705aa670)

Reply:
![image](https://github.com/chevyB/pcbee/assets/89174822/58c4474f-5675-41c9-b9b6-b97debdf5e53)